browser/ui/commands/accelerator_service.cc

```diff
@@ -29,10 +29,13 @@ namespace {
 
 mojom::CommandPtr ToMojoCommand(
     int command_id,
-    const std::vector<ui::Accelerator>& accelerators) {
+    const std::vector<ui::Accelerator>& accelerators,
+    const std::vector<ui::Accelerator>& default_accelerators) {
   auto command = mojom::Command::New();
   command->id = command_id;
   command->name = std::string(commands::GetCommandName(command_id));
+  command->modified = accelerators != default_accelerators;
```

Comment:
For example, 
When `default_accelerators` is ["cmd+a", "cmd+b"]
and `accelerators` is ["cmd+b", "cmd+a"].
Curious `modified` should be `false` or not in this case?

Reply:
Good point! Fixed